Code/Framework/AzToolsFramework/AzToolsFramework/aztoolsframework_files.cmake

```diff
@@ -657,6 +657,9 @@ set(FILES
     Prefab/PrefabPublicHandler.cpp
     Prefab/PrefabPublicInterface.h
     Prefab/PrefabPublicNotificationBus.h
+    Prefab/PrefabPublicRequestBus.h
+    Prefab/PrefabPublicRequestHandler.h
+    #Prefab/PrefabPublicRequestHandler.cpp
```

Comment:
commented file? perhaps leftovers of old local code?

Reply:
Will fix it.

Reply:
Fixed.